v20.1/authorization.md

```diff
@@ -2,7 +2,7 @@
 title: Authorization
 summary: Learn about the authorization features for secure CockroachDB clusters.
 toc: true
-redirect_from: create-and-manage-users.html
+redirect_from: [create-and-manage-users.html, roles.html]
```

Comment:
Will that formatting work? I'm not sure. I see it handled like this elsewhere:

```
redirect_from:
- back-up-data.html
- restore-data.html
```

Reply:
Nevermind. Your approach works and matches:

```
redirect_from: [show-testing-ranges.html, show-experimental-ranges.html]
```

Reply:
Oh, interesting. I referred to the formatting in the `show ranges`doc:
~~~
redirect_from: [show-testing-ranges.html, show-experimental-ranges.html]
~~~